pkg/agent/datamodel/types.go

```diff
@@ -2173,6 +2180,9 @@ func (err *CSEStatusParsingError) Error() string {
 
 type AgentPoolWindowsProfile struct {
 	DisableOutboundNat *bool `json:"disableOutboundNat,omitempty"`
+
+	// Windows next-gen networking uses Windows eBPF for the networking dataplane.
+	NextGenNetworkingURL string `json:"nextGenNetworkingURL,omitempty"`
```

Comment:
Using `string` here to match the pattern for other components (`WindowsCalicoPackageURL`, `WindowsGmsaPackageUrl`, `GpuDriverURL`)

Reply:
switched back to *string